packages/fuels-abigen-macro/tests/test_projects/auth_testing_contract/src/main.sw

```diff
@@ -14,12 +14,12 @@ impl AuthTesting for Contract {
     }
 
     fn check_msg_sender(expected_id: Address) -> bool {
-        let result: Result<Identity, AuthError> = msg_sender();
+        let response: Result<Identity, AuthError> = msg_sender();
```

Comment:
Correct me if I'm wrong @vnepveu, but I think the original (poorly-documented) reasoning of the issue was to change "result" to "response" for contract calls, since it's over the network (local, but still).

This one here isn't a response, because there's no network request. It's actually a result.

Reply:
Yup, that's what I mentioned to @thdaraujo in other channels; this refactor should only be related to variables receiving the response from a network call. So, everything with a `.call()` or `.transfer()` in this case.